README.md

```diff
@@ -1,24 +1,29 @@
 # manage-recalls-ui
 UI for managing recalls
 
-## Running the app
-The easiest way to run the app is to use docker compose to create the service and all dependencies. 
-
-`docker-compose pull`
-
-`docker-compose up`
-
 ### Dependencies
-The app requires: 
+The app requires:
 * hmpps-auth - for authentication
 * redis - session store and token caching
 * manage-recalls-api - api service for managing the recall process
 
+## Running the app
+The easiest way to run the app is to use docker compose to create the service and all dependencies. This starts the latest published docker container for hmpps-auth and redis, a fake manage-recalls-api (wiremock) and a local build of the manage-recalls-ui. 
+
+`docker compose up`
```

Comment:
I think this is a firewall issue locally for me but at present I cannot reach `localhost:3000` from Chrome after starting docker as here.

Reply:
We probably need to debug this on your machine and see what's going on